docarray/array/storage/elastic/find.py

```diff
@@ -31,21 +29,51 @@
 
 
 class FindMixin(BaseFindMixin):
-    def _find_similar_vectors(self, query: 'ElasticArrayType', limit=10):
+
+    def _find_similar_vectors(self, query: 'ElasticArrayType', limit=10, **kwargs,
```

Comment:
add the parameter `filter_filed`, `filter_values` in the function signature, and add docstring.

Reply:
reuse `filter` parameter in `find()` method